subsys/bluetooth/controller/ll_sw/ull_llcp_common.c

```diff
@@ -235,7 +235,14 @@ static void lp_comm_tx(struct ll_conn *conn, struct proc_ctx *ctx)
 		 * NOTE: As the supervision timeout is at most 32s the normal procedure response
 		 * timeout of 40s will never come into play for the ACL Termination procedure.
 		 */
-		const uint32_t conn_interval_us = conn->lll.interval * CONN_INT_UNIT_US;
+		uint32_t conn_interval_us;
+
+		if (conn->lll.interval >= BT_HCI_LE_INTERVAL_MIN) {
+			conn_interval_us = conn->lll.interval * CONN_INT_UNIT_US;
+		} else {
+			conn_interval_us = (conn->lll.interval + 1U) * CONN_LOW_LAT_INT_UNIT_US;
+		}
```

Comment:
There should a comment stating why/how/when `if (conn->lll.interval >= BT_HCI_LE_INTERVAL_MIN)` is true, and why, when it is true, the calculation changes from the intuitive `conn_interval_us = conn->lll.interval * CONN_INT_UNIT_US;` to something that is seemingly completely different

Reply:
These change are base dependencies for now adopted Frame Space Update procedure in PR: https://github.com/zephyrproject-rtos/zephyr/pull/82324

I am hoping the non-standard `CONFIG_BT_CONN_PARAM_ANY` can be deprecated in the future. I can discuss offline on the details with you.

Reply:
In any case, this code is non-standard (not spec-compliant) and pretty undocumented. We should add comment stating why this is here, and why it's `interval + 1U` and why `CONN_LOW_LAT_INT_UNIT_US` is used instead of `CONN_INT_UNIT_US`